opba-protocols/xs2a-protocol-tests/xs2a-bdd-tests-common/src/main/java/de/adorsys/opba/protocol/xs2a/tests/e2e/stages/AccountInformationRequestCommon.java

```diff
@@ -416,12 +413,11 @@ protected void updateRedirectCode(ExtractableResponse<Response> response) {
 
     @SneakyThrows
     private void updateAvailableScas() {
-        ExtractableResponse<Response> response = RestAssured
+        ExtractableResponse<Response> response = withSignatureHeaders(RestAssured
```

Comment:
This function is called within the user session (from UI), not Fintech - it is protected only by users' cookie and not by request signing - no signature headers here

Reply:
Removed signature header adding for non-fintech requests